src/main/java/com/icthh/xm/tmf/ms/communication/messaging/handler/TemplatedEmailMessageHandler.java

```diff
@@ -59,14 +63,15 @@ public CommunicationMessage handle(CommunicationMessageCreate messageCreate) {
             .collect(Collectors.toList());
 
         Map<String, Object> objectModel = toObjectModel(messageCreate.getCharacteristic());
+        Map<String, Object> templateModel = getTemplateModel(objectModel);
         String language = String.valueOf(objectModel.get(LANGUAGE));
         String templateName = String.valueOf(objectModel.get(TEMPLATE_NAME));
         Locale locale = new Locale(language);
         String sender = messageCreate.getSender().getId();
         String subject = messageCreate.getSubject();
 
         for (String email : emails) {
-            sendWithAttachments(objectModel, templateName, locale, sender, subject, email, messageCreate.getAttachment());
+            sendWithAttachments(templateModel, templateName, locale, sender, subject, email, messageCreate.getAttachment());
```

Comment:
I guess it should not broke existing email because I've added condition that if objectModel doesn't have templateModel key it should return this objectModel.
The piece of these changes: 
``` 
   private Map<String, Object> getTemplateModel(Map<String, Object> objectModel) {
        Object templateModel = objectModel.get(TEMPLATE_MODEL);

        if (templateModel == null) {
            return objectModel;
        }
```

Reply:
+ all existing tests that cover sending mails via MailService passed